test/test.ml

```diff
@@ -1,16 +1,15 @@
 open OUnit
+open Result
 
 let redis_test_host () =
   try
     Sys.getenv("OCAML_REDIS_TEST_IP")
   with Not_found ->
     "127.0.0.1"
 
-let redis_test_port () =
-  try
-    int_of_string(Sys.getenv("OCAML_REDIS_TEST_PORT"))
-  with Not_found ->
-    6379
+let redis_test_port = 63791
```

Comment:
why not keep the original code and use env variables in the test setup?

Reply:
The main reason was I'm now using 3 ports:

- 63791 for the redis install with no auth
- 63792 for the redis install with auth
- 63793 for the redis install with acl

Would you prefer to have 3 separate env variables that can be configured? I don't think it's possible to just have one env variable, because some of the tests need to communicate with different Redis instances.

Reply:
maybe the ACL tests can live in another file, which means another test?

although, you're right, this is just a test file anyway, it's ok to hardcode values. Carry on.